src/diffDOM/dom/apply.js

```diff
@@ -49,7 +49,7 @@ export function applyDiff(
             }
             node.setAttribute(diff[options._const.name], diff[options._const.newValue])
             if (node.nodeName === 'INPUT' && diff[options._const.name] === 'value') {
-                node.value = diff[options._const.oldValue]
```

Comment:
@z1ad That was actually done on purpose back then. The idea was that if the node's value attribute had changed, but it's value had not changed, we needed to reset the value to what it had been previously after setting the attribute.

This code may be outdated now or not working with latest browser implementations. In that case we probably need to remove the entire if-block.

Reply:
It may have to do with this: https://github.com/fiduswriter/diffDOM/issues/73